mwa_trigger/trigger_logic.py

```diff
@@ -30,17 +30,84 @@ def worth_observing(voevent, max_duration=2.05, fermi_prob=50):
         if voevent.most_likely_index == 4:
             logger.debug("MOST_LIKELY = GRB")
             # ignore things that don't reach our probability threshold
-            if voevent.detect_prob  > fermi_prob:
+            if voevent.detect_prob > fermi_prob:
                 trigger_bool = True
-                trigger_message += f"Fermi GRB probabilty greater than {fermi_prob}. "
+                trigger_message += f"Fermi GRB probability greater than {fermi_prob}. "
             else:
                 debug_bool = True
-                trigger_message += f"Fermi GRB probabilty less than {fermi_prob}. "
+                trigger_message += f"Fermi GRB probability less than {fermi_prob}. "
         else:
             logger.debug("MOST LIKELY != GRB")
             debug_bool = False
-            trigger_message += f"Fermi GRB likey index not 4. "
+            trigger_message += f"Fermi GRB likely index not 4. "
     elif short_bool:
         trigger_bool = True
 
-    return trigger_bool, debug_bool, short_bool, trigger_message
\ No newline at end of file
+    return trigger_bool, debug_bool, short_bool, trigger_message
+
+
+def worth_observing_atca_short_grb(voevent):
+    """
+    Decide if an event is worth observing as part of the ATCA sGRB project
+
+    Parameters
+    ----------
+    voevent : :obj:`mwa_trigger.parse_xml.parsed_VOEvent`
+      The VOEvent to consider.
+
+    Return
+    ------
+    trig, hold, msg : (bool, bool, string)
+       The trigger status, True = worth observing.
+       If the trigger should be held for manual intervation, True = on hold.
+       A message detailing the decision logic.
+    """
+    # Setup up defaults
+    trigger = False
+    hold = False
+    trigger_message = ""
+
+    # TODO: Move these to a table in the database
+    MAGNETAR_LIMIT = 0.033
+    SHORT_LIMIT = 0.257
```

Comment:
Turn these into optional parameters of the function and make these the defaults. This will make it easier for the database to edit them.

Reply:
will do

Reply:
done in dabdd2c